dallinger/heroku/tools.py

```diff
@@ -164,7 +164,16 @@ def name(self):
 
     @property
     def url(self):
-        return "https://{}.herokuapp.com".format(self.name)
+        app_info = self._result(
+            [
+                "heroku",
+                "apps:info",
+                "--app",
+                self.name,
+                "--json",
+            ]
+        )
+        return json.loads(app_info)["app"]["web_url"]
```

Comment:
Maybe for back-compatibility we should strip the trailing `/` from this URL?

Reply:
:+1:  done